src/tasks/components/TaskHeaderCard.tsx

```diff
@@ -0,0 +1,174 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+import {withRouter, RouteComponentProps} from 'react-router-dom'
+import Moment from 'react-moment'
+
+// Components
+import {
+  SlideToggle,
+  ComponentSize,
+  ResourceCard,
+  InputLabel,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Actions
+import {
+  addTaskLabel,
+  deleteTaskLabel,
+  runTask,
+  getRuns,
+} from 'src/tasks/actions/thunks'
+
+// Types
+import {ComponentColor, Button} from '@influxdata/clockface'
+import {Task, AppState} from 'src/types'
+
+interface PassedProps {
+  task: Task
+  onActivate: (task: Task) => void
+  onEditTask: () => void
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = PassedProps & ReduxProps
+
+export class TaskHeaderCard extends PureComponent<
+  Props & RouteComponentProps<{orgID: string; id: string}>
+> {
+  public render() {
+    const {task} = this.props
+
+    if (!task) {
+      return null
+    }
+
+    return (
+      <ResourceCard
+        testID="task-card"
+        disabled={!this.isTaskActive}
+        alignItems={AlignItems.Center}
+        margin={ComponentSize.Large}
+        direction={FlexDirection.Row}
+        justifyContent={JustifyContent.SpaceBetween}
+      >
+        <FlexBox
+          alignItems={AlignItems.FlexStart}
+          direction={FlexDirection.Column}
+          margin={ComponentSize.Large}
+        >
+          <ResourceCard.Name name={task.name} testID="task-card--name" />
+          <ResourceCard.Meta>
+            {this.activeToggle}
+            <>Created at: {task.createdAt}</>
+            <>Created by: {task.name}</>
+            <>
+              Last Used:{' '}
+              <Moment fromNow ago>
```

Comment:
we shouldn't be introducing a dependency for this. The time can be formatted using `moment`, but I believe there's an epic in the backlog to remove `Moment` as a dependency as a whole. Regardless I'd err on using moment.js to make this happen rather than a new dependency